substrafl/exceptions.py

```diff
@@ -14,6 +14,10 @@ class UnsupportedPythonVersionError(Exception):
     """The Python version used is not supported by Substra."""
 
 
+class UnsupportedClientBackendTypeError(Exception):
+    """The client backend type is not supported."""
```

Comment:
Nitpick but isn't it more that the backend type is not supported for this given mode / function?